### DISCUSSION TITLE
Gmail schedule send "shortcuts"

### DISCUSSION THREAD

**Discussion by andershoejlund** [Extended-Thunder/send-later#298] [Ideas]
Hi,

Thank you for a great thunderbird add-on.

I have used schedule send a lot on gmail and like the shortcuts:
- Tomorrow Morning - 8:00
- Tomorrow Afternnon - 13:00
- Monday Morning - 8:00

In your prefences, under shortcuts, I am able to, as you of course know, customize the delay, setup days in a row, business hours (not sure what it does?), how can I make the same settings as the gmail default?

Thank you very much for your time and effort. best regards Anders

**Comment by jikamens** [comment]
Please see the documentation for dynamic scheduling functions [in the user guide](https://extended-thunder.github.io/send-later/#dynamic-functions-for-complex-scheduling-rules).
